src/app/main/component/events/components/event-editor/components/create-event-information/create-event-information.component.ts

```diff
@@ -40,21 +40,17 @@ export class CreateEventInformationComponent implements OnInit {
   }
 
   quillContentChanged(content: ContentChange): void {
-    const quill = content.editor;
-    const range = quill.getSelection();
-
+    // No form update here, just capturing typing behavior
```

Comment:
you do not need to leave a comment here

I can ask you to look at create-eco-news if it works the same way, because we are also using the same one
thank you,

Reply:
> you do not need to leave a comment here
> 
> I can ask you to look at create-eco-news if it works the same way, because we are also using the same one thank you,

removed comment

It works different at create-eco-news, however, we get error there on every change and form value is changed with every symbol inputed